spdx-utils/build.gradle.kts

```diff
@@ -214,6 +218,11 @@ fun generateEnumClass(
     |         * - has intentionally provided no information (no meaning should be implied by doing so).
     |         */
     |        const val NOASSERTION = "NOASSERTION"
+    |
+    |        /**
+    |         * The version of the license list.
+    |         */
+    |        const val LICENSE_LIST_VERSION = "$licenseListVersion"
```

Comment:
How about making this the first constant, before the license ids?

Reply:
@mnonnenmacher  @sschuberth  - Do you guys want me to move the whole companion object block upwards?

Reply:
No, I just thought the order of constants should be
- `LICENSE_LIST_VERSION`
- `NONE`
- `NOASSERTION`

Reply:
I've moved it above the other two constants for now.